### PR TITLE
Migrate from READMEs to RTD site

### PR DESCRIPTION
**Pull request by parente**
Simplify READMEs and point to the RTD site for details. Move the maintainer notes to the RTD site as well.

More work for #517.

### CODE REVIEW

**Comment by parente** [comment]
@minrk @willingc @jzf2101 I've added your GitHub usernames as admins to https://readthedocs.org/projects/jupyter-docker-stacks/ to increase the bus factor. If your GitHub username is not your RTD username, let me know and I'll correct that.

File: README.md [hunk 1]
```diff
@@ -1,134 +1,32 @@
...
-# -rm: remove the container on exit.
-# -p: publish port to the host
-docker run -ti --rm -p 8888:8888 jupyter/<your desired stack>:<git-sha-tag>
+The [User Guide on ReadTheDocs](http://jupyter-docker-stacks.readthedocs.io/) describes additional uses and features in detail.
 
-# Run a Jupyter Notebook server in a Docker container in the terminal foreground.
-# Any files written to ~/work in the container will be saved to the current working
-# directory on the host.
-docker run -ti --rm -p 8888:8888 -v "$PWD":/home/jovyan/work jupyter/<your desired stack>:<git-sha-tag>
+**Example 1:** This command pulls the `jupyter/scipy-notebook` image tagged `2c80cf3537ca` from Docker Hub if it is not already present on the local host. It then starts a container running a Jupyter Notebook server and exposes the server on host port 8888. The server logs appear in the terminal and include a URL to the notebook server. The container remains intact for restart after notebook server exit.::
```
Comment (willingc) [inline README.md]: If markdown, the double colon at end is unnecessary. One can be removed.

File: README.md [hunk 2]
```diff
@@ -1,134 +1,32 @@
...
-# Run an ephemeral Jupyter Notebook server in a Docker container in the background.
-# Note that any work saved in the container will be lost when it is destroyed with this config.
-# -d: detach, run container in background.
-# -P: Publish all exposed ports to random ports
-docker run -d -P jupyter/<your desired stack>:<git-sha-tag>
-```
+    docker run -p 8888:8888 jupyter/scipy-notebook:2c80cf3537ca
 
-## Getting Started
+**Example 2:** This command pulls the `jupyter/r-notebook` image tagged `e5c5a7d3e52d` from Docker Hub if it is not already present on the local host. It then starts an *ephemeral* container running a Jupyter Notebook server and exposes the server on host port 10000. The command mounts the current working directory on the host as `/home/jovyan/work` in the container. Docker destroys the container after notebook server exit, but any files written to `~/work` in the container remain intact on the host.::
```
Comment (willingc) [inline README.md]: Same re: colons

File: README.md [hunk 3]
```diff
@@ -1,134 +1,32 @@
...
-If this is your first time using Docker or any of the Jupyter projects, do the following to get started.
+    docker run --rm -p 10000:8888 -v "$PWD":/home/jovyan/work jupyter/r-notebook:e5c5a7d3e52d
 
-1. [Install Docker](https://docs.docker.com/installation/) on your host of choice.
-2. Open the README in one of the folders in this git repository.
-3. Follow the README for that stack.
+## Contributing
 
-## Visual Overview
+Please see the [Contributor Guide on ReadTheDocs](http://jupyter-docker-stacks.readthedocs.io/) for information about how to contribute package updates, recipes features, tests, and community maintained stacks.
```
Comment (willingc) [inline README.md]: recipes' features

File: README.md (hunk 1 above)
Comment (parente) [reply README.md]: Good catch. Copy/paste error from the index.rst on the docs site.

**Comment by willingc** [comment]
Thanks @parente. Please double check that the docs serve match your expectation. :sunny:

**Comment by parente** [comment]
Thanks for the review and merge, @willingc. The live RTD site looks as I would expect at the moment.

More PRs to come ...

**Pull request merged**